### PR TITLE
[Sprint Maintain] Feature fix ui dialog error leave request

### PR DESCRIPTION
**Pull request by HyperionsThs**
Link : None

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -240,8 +240,8 @@
     <string name="the_number_of_days_allowed">The number of days allowed for vacations may not be left blank!</string>
     <string name="the_field_required_can_not_be_blank">The field required can not be blank!</string>
     <string name="reason_can_not_empty">Reason can not be blank!</string>
-    <string name="datetime_have_salary_can_not_empty">Date time have salary can not be blank!</string>
-    <string name="datetime_no_salary_can_not_empty">Date time no salary can not be blank!</string>
+    <string name="datetime_have_salary_can_not_empty"><![CDATA[Date time<b> have salary</b> can not be blank! ]]></string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`datetime_have_salary_can_not_empty`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -240,8 +240,8 @@
...
+    <string name="datetime_no_salary_can_not_empty"> <![CDATA[Date time<b> no salary</b> can not be blank! ]]></string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`datetime_no_salary_can_not_empty`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/layout/fragment_time_sheet.xml [hunk 3]
```diff
@@ -256,9 +256,11 @@
                                 android:layout_width="match_parent"
                                 android:layout_height="match_parent"
                                 android:gravity="center_vertical"
-                                android:padding="@dimen/dp_10"
+                                android:paddingBottom="@dimen/dp_10"
+                                android:paddingLeft="@dimen/dp_10"
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/fragment_time_sheet.xml]: WARNING: [RtlSymmetry] When you define `paddingLeft` you should probably also define `paddingRight` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.

File: app/src/main/res/layout/fragment_time_sheet.xml [hunk 4]
```diff
@@ -256,9 +256,11 @@
...
+                                android:paddingStart="@dimen/dp_10"
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/fragment_time_sheet.xml]: WARNING: [RtlSymmetry] When you define `paddingStart` you should probably also define `paddingEnd` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.

File: app/src/main/java/com/framgia/wsm/screen/BaseActivity.java [hunk 5]
```diff
@@ -20,4 +22,9 @@ protected void onStop() {
         EventBus.getDefault().unregister(this);
         super.onStop();
     }
+
+    @Override
+    public void onBackPressed() {
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/BaseActivity.java]: ?

Comment (HyperionsThs) [reply app/src/main/java/com/framgia/wsm/screen/BaseActivity.java]: bữa trước a có nói e làm animation khi bấm back ở 3 cái request, mà 3 cái request extent thằng này để e ghi chắc thằng này để 3 cái kia nhận luộn ạ (bow)

Comment (daolq3012) [reply app/src/main/java/com/framgia/wsm/screen/BaseActivity.java]: Như này thì tất cả các activity kế thừa từ base đều có animation (tat) activity nào thì overide activity đó

**Pull request merged**